### DISCUSSION TITLE
Gulp plugin with more recent versions of gradle-node-plugin (can't find support after 1.5.3)

### DISCUSSION THREAD

**Discussion by dvuke** [node-gradle/gradle-node-plugin#298] [Q&A] [answered]
Hey everyone,

Just had a quick question; I was using the moo works plugins for node/gulp, and stumbled upon this project for forward versions of gradle support.

I was able to track down some documentation in tags here, and found that version 1.5.3 of the gradle-node-plugin had a nice gulp plugin integration to easily use.  I got that to work fine.  I started to look at more recent versions of the plugin, however, 2.x+ and couldn't find anything dealing with how to integrate gulp with the node plugin.

Can anyone lend a hand here? 

Thanks,
Damian

**Comment by dvuke** [comment]
Any input on this @deepy ?

**Comment by deepy** [comment, marked as answer]
We've got a section for this in the [FAQ](https://github.com/node-gradle/gradle-node-plugin/blob/main/docs/faq.md#gulp)
The short of it is that `NpxTask` should be covering all that was previously handled by the grunt and the gulp tasks

If you find some use-case that's not covered we'd love to hear about it though!
